### PR TITLE
Redis Configuration

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- Redis Configuration
- Business Short Code Configuration

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj @avikganguly01

**Review by fynmanoj — CHANGES_REQUESTED** [review]
I could see a master/slave ( 2 nodes) came alive for redis in SIT, can you change this to single Node ? we need Bare minimus setup as default

File: helm/ph-ee-engine/values.yaml [hunk 1]
```diff
@@ -921,3 +921,8 @@ keycloak:
   extraEnv: |
     - name: KEYCLOAK_IMPORT
       value: /realm/kong-keycloak-realm.json
+
+redis:
+  enabled: false
+  replica:
+    replicaCount: 1
```
[the remark this answers is not in the document]
Comment (fynmanoj) [reply helm/ph-ee-engine/values.yaml:928]: we don't need two nodes for the bare minimum setup. use disable options to disable one node

[the remark this answers is not in the document]
Comment (truthfool) [reply helm/ph-ee-engine/values.yaml]: Done

[the remark this answers is not in the document]
Comment (fynmanoj) [reply helm/ph-ee-engine/values.yaml]: is it tested?

**Pull request merged**